src/xcode/ENA/ENA/Source/Services/TicketValidation/AllowList/SAP_Internal_Dgc_ValidationServiceAllowlist+Extension.swift

```diff
@@ -3,6 +3,7 @@
 //
 
 import Foundation
+import ENASecurity
```

Comment:
Is this needed, as there are not other changes to this file?

Reply:
the ValidationServiceAllowlistEntry struct is moved into the security package as it is needed for the Trust Evaluation class